apps/kg-explorer/src/EMBEDDING.md

```diff
@@ -0,0 +1,23 @@
+# Embedding _Knowledge Graph_
+
+The API endpoint and mirror url can be set as in this example:
+
+```html
+<!doctype html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <title>HRA Knowledge Graph Explorer</title>
+    <base href="/" />
```

Comment:
@edlu77  @axdanbol  Please provide an example where the base href is on another site (ie the mirror) and the code is deployed to cdn.